apps/channels/clients/connect_client.py

```diff
@@ -0,0 +1,31 @@
+import logging
+from uuid import UUID
+
+import httpx
+import requests
+from django.conf import settings
+from tenacity import before_sleep_log, retry, retry_if_exception_type, stop_after_attempt, wait_exponential
+
+logger = logging.getLogger("connectid-api")
+
+
+class CommCareConnectClient:
+    def __init__(self):
```

Comment:
Maybe also check if these settings are all set and raise an exception if not

Reply:
https://github.com/dimagi/open-chat-studio/pull/1035/commits/b58eb00b04848bb05dc5f571ff2bec443ac4ad83

Since we default to a value for `CONNECT_ID_SERVER_BASE_URL`, I didn't check that